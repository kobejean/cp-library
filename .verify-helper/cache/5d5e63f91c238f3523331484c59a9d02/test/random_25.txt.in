98
658040077 60892294 475675815
580025759 386763334 889364881
795293383 511365171 728656811
777647102 286902350 250711411
443496871 240802405 616050005
776846582 126066348 867137875
624162677 340337498 419719782
598818586 515032452 672352539
954270341 216222138 535433012
670413027 624510490 437807381
204219671 66365401 439536973
734241066 501932926 573585007
155263897 37551369 969732094
136240069 59450524 30291294
614105389 224497085 579494572
861952509 635415687 434021524
755081 88821 210963710
514781038 49724570 101477551
475720117 376553522 8537645
990418857 837298803 702632531
351903353 340556145 866745489
973093453 794029831 112932394
627024407 527324729 740868117
811582702 433883505 407012749
626504057 315617164 922378968
295621902 121050967 470192837
954217709 232202284 793700902
478941319 397204855 324405183
806986681 219224933 920380654
247606520 75874486 939483721
558449053 461535770 435363122
351577467 346219882 549406762
385661513 176804010 379128337
963437207 651800238 800351875
188524009 183432442 475253551
256350019 173645150 456810211
291553261 68414425 542053165
531779646 253924647 493985309
612159761 26625621 306483299
883272952 222733676 493258487
32277731 14294893 641673568
622516467 308813972 87464995
721732933 248415039 186776691
990679730 946905721 131122313
675876931 166781217 975047697
223857723 102555635 565061071
475155469 49404916 686407808
89328439 32601098 957143289
88924974 23615946 908157627
191270145 18423858 15421698
876680230 686569388 23397348
43799939 31978649 254566177
338480647 63958220 677837434
159749657 150986595 328062287
213111903 96203250 469318782
347224901 299524943 856414231
91300256 81251347 155256328
167852426 46317810 430297158
995427851 85503572 766239155
965399300 668403085 146326065
990592260 785876493 979523577
300734765 245311600 153883835
502960564 392489400 271117698
271407360 146927772 303614238
531436141 531228858 992000311
389181677 249889072 389181677
794340734 762217256 794340734
455899963 120745832 455899963
286384704 85710166 859154112
213347129 192886782 853388516
209968650 17840785 629905950
586538177 129738885 586538177
466871004 443055877 466871004
953124563 866390621 953124563
252566931 194396972 505133862
514757317 131783658 514757317
222741117 181450809 445482234
374467231 203904361 374467231
562287776 321176632 562287776
249081577 73058770 249081577
999999999 703703703 209876543
999999925 957894665 378947340
999999918 319148910 308510613
999999982 447368413 26315789
999999982 368421046 105263156
999999984 580645152 725806440
1000000000 500000000 375000000
1000000000 937500000 78125000
1000000000 200000000 550000000
999999988 423076918 884615374
1000000000 500000000 234375000
999999984 580645152 32258064
999999965 148936165 595744660
1000000000 600000000 810000000
999999980 499999990 423913035
999999975 706666649 293333326
999999990 533333328 366666663
999999991 447761190 895522380
