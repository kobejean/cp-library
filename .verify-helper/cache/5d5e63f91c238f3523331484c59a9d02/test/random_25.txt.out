176378505
10
795293331
321356660
68
776846572
108083138
50
954270280
231157807
11
734240996
109070098
70
614105360
656745939
15
514781002
315581666
27
351903271
812278130
89
811582679
290787781
1
954217651
442989635
55
247606434
463757775
80
385661422
88871723
73
256349990
247458227
63
612159695
667022308
12
622516435
324869722
23
675876883
52138750
47
-1
74
63756689
96710409
89
548518
-1
67
892590
-1
35
90493424
177442691
71
60146926
229922390
46
10027082
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
30
74
80
21
6
24
4
52
16
21
32
13
35
40
46
1
4
33
